nntrainer/tensor/manager.h

```diff
@@ -538,6 +542,33 @@ class Manager {
    * @brief Initialize the tensors for training mode
    */
   void initializeTensorsTrain();
+
+  /**
+   * @brief     Create tensors with the given spec
+   *
+   * @param w   node Graph node to extract node identifiers/info
+   * @param w   create tensors list
+   * @param layer_objs_list list to store the created tensors
+   */
+  template <typename T>
+  static std::vector<T *> _requestTensors(
```

Comment:
Could you check this?

s/_requestTensors/requestTensors_/

Reply:
Could you also update the docs?

Reply:
did you mean _requestTensors -> requestTensors?

Reply:
the github markdown messed up the comment 🤣  `s/_requestTensors/requestTensors_/`

Reply:
updated  `s/_requestTensors/requestTensors/`